applications/nrf_desktop/doc/ble_scan.rst

```diff
@@ -140,9 +142,8 @@ a. The scanning is stopped and the |NCS|'s :ref:`nrf_bt_scan_readme` automatical
    The initial Bluetooth connection interval is set by default to 7.5 ms, that is to the shortest connection interval allowed by the Bluetooth specification.
 #. The peer discovery is started.
 #. After the :ref:`nrf_desktop_ble_discovery` completes the peer discovery, the :ref:`nrf_desktop_ble_conn_params` receives the ``ble_discovery_complete_event`` and updates the Bluetooth connection interval.
+   At this point, the scanning can be restarted.
```

Comment:
```suggestion
   At this point, you can restart the scanning.
```
Changed as it's a part of the instruction

Reply:
This is more like information what happens (the dongle just does it, user does not need to perform additional steps here).

Reply:
got it, please disregard the suggestion:)